src/SecurityManager.ts

```diff
@@ -341,22 +341,9 @@ export async function withSecretStorageKeyCache<T>(func: () => Promise<T>): Prom
  * @param {Function} [func] An operation to perform once secret storage has been
  * bootstrapped. Optional.
  * @param {bool} [forceReset] Reset secret storage even if it's already set up
- * @param {bool} [setupNewKeyBackup] Reset secret storage even if it's already set up
  */
-export async function accessSecretStorage(
-    func = async (): Promise<void> => {},
-    forceReset = false,
-    setupNewKeyBackup = true,
-): Promise<void> {
-    await withSecretStorageKeyCache(() => doAccessSecretStorage(func, forceReset, setupNewKeyBackup));
-}
```

Comment:
This change was introduced in https://github.com/matrix-org/matrix-react-sdk/pull/12059. I'd suggest leaving it in place (without `setupNewKeyBackup`, of course).

Reply:
Done https://github.com/matrix-org/matrix-react-sdk/pull/12102/commits/35806b5bc793b233011ceeccbf8cd09fba6745a5